firebase.json

```diff
@@ -1,11 +1,7 @@
 {
   "hosting": {
     "public": "dist",
-    "ignore": [
-      "firebase.json",
-      "**/.*",
-      "**/node_modules/**"
-    ],
+    "ignore": ["firebase.json", "**/.*", "**/node_modules/**"],
```

Comment:
Any idea why these got collapsed? Is it a settings change from prettier or so? I personally prefer the old style.

Reply:
Nevermind, they are on one line, not two as it seemed in the diff view. Still looks good 👍